src/sentry/issues/occurrence_consumer.py

```diff
@@ -136,8 +139,16 @@ def _process_message(
 
     metrics.incr("occurrence_ingest.messages", sample_rate=1.0)
 
+    # only validating event data, for now
     if "event_data" in kwargs:
-        return process_event_and_issue_occurrence(**kwargs)  # returning this now for easier testing
+        try:
+            schema_version: int = kwargs["event_data"].get("version", 0)
+            jsonschema.validate(kwargs["event_data"], EVENT_PAYLOAD_VERSIONS[schema_version])
+        except jsonschema.ValidationError:
+            metrics.incr("occurrence_ingest.message_wrapper_invalid")
+            raise Exception("Message wrapper does not match schema")
```

Comment:
Could we add a test when this exception gets raised.

Reply:
definitely